server/controllers/user.js

```diff
@@ -203,12 +245,10 @@ async function getInfos(req, res) {
  * @param {HttpResponseWrapper} res The http response
  */
 async function saveInfos(req, res) {
-  let user = String(req.params.pseudo);
-  if (user === "me") {
-    user = req.body.authUser;
-  }
+  const param = String(req.params.pseudo);
+  const user = param === "me" ? req.body._auth.user : param;
 
-  if (req.body.authUser !== user) {
+  if (req.body._auth.user !== user) {
     // TODO? Add admin ?
```

Comment:
And admin should have the rights (I think) to update another one's profile. If you disagree you can delete the //TODO, or if we want to to this later, we can create an issue to keep track of this :D

Reply:
I think we can remove this one !